nengo/utils/numpy.py

```diff
@@ -161,7 +161,9 @@ def rms(x, axis=None, keepdims=False):
         If True, the reduced axes are left in the result. See `np.sum` in
         newer versions of Numpy (>= 1.7).
     """
-    return np.sqrt(np.mean(x**2, axis=axis, keepdims=keepdims))
+
+    x = np.asarray(x)
+    return np.sqrt(np.mean(x ** 2, axis=axis, keepdims=keepdims))
 
 
 def rmse(x, y, axis=None, keepdims=False):
```

Comment:
Unrelated, but I really hate that I ever made this function. It's so easy just to do `rms(x - y)`, there's no reason to even have it. Could we deprecate it?